### ISSUE TITLE
Enhancement: Registerd Studentlist

### ISSUE THREAD

**Issue by Dirowa** [galaxyproject/tiaas2#66]
Hi, at Avans University of applied science, do we use the TIAAS to give galaxy classes so students learn to analyse their own data they generate. 
We love this features, but some teachers are asking if it would be possible to see a list of students who subscribed to the TIAAS?
Could this be possible to keep track of a participant list?

**Comment by hexylena** [comment]
So you can see student names in the 'status' page right? I added a flag for that specifically for avans so that should be there. But you want to see a list of which students are there as well? (not identified by looking at the jobs list?)

**Comment by Dirowa** [comment]
Thats correct and we think it is quite usefull to see where it is going wrong.

But we currently are holding the galaxy classes online due lack of computer rooms. But not every student is attending. We suspect that they are just in the teams call and are going back to bed. so what we are thinking is something like this. more to use as an attendance list, perhaps with an overview howmuch they participated?

**Comment by Dirowa** [comment]
That is definitly a option, then we will do that

**Comment by hexylena** [comment]
We also have that information, which is exposed as Avans sets that flag, so if a user list is better we can expose that, I can show ya where to set it.
